dubbo-container/dubbo-container-spring/src/main/java/org/apache/dubbo/container/spring/SpringContainer.java

```
@@ -48,8 +48,16 @@ public void start() {
             configPath = DEFAULT_SPRING_CONFIG;
         }
         context = new ClassPathXmlApplicationContext(configPath.split("[,\\s]+"), false);
-        context.refresh();
-        context.start();
+        try {
+            context.refresh();
+        } catch (Throwable e) {
+            logger.error("refresh spring container failed!", e);
```

Should we throw the exception again?

---

If we swallow the exception, users could not know what happends.